pkg/autoscaler/metrics/collector.go

```diff
@@ -299,12 +328,18 @@ func newCollection(metric *autoscalingv1alpha1.Metric, scraper StatsScraper, clo
 		defer c.grp.Done()
 
 		scrapeTicker := clock.NewTicker(scrapeTickInterval)
+		var pause bool
 		defer scrapeTicker.Stop()
 		for {
 			select {
 			case <-c.stopCh:
 				return
+			case pause = <-c.pauseCh:
 			case <-scrapeTicker.C():
+				if pause {
+					continue
+				}
+
```

Comment:
Currently, there's only coverage to pause, I assume based on TBC==-1. There's no logic to resume scrapping, correct?